lib/common/email.sh

```diff
@@ -165,7 +166,11 @@ get_uploader_email() {
             return 0
         fi
     fi
-    log_error "Could not find uploader for file '$file'"
+
+    local error_msg = "Could not find uploader for file '$file'"
+    log_error "'$error_msg'"
```

Comment:
Not sure this functionality is needed. 1) When the uploader is not fund, the email could go to the backup recipient but not the whole group or POs are going to be inundated with emails. 2) each time POs  push a file through the pipeline we will receive an email: last week i re-pushed 150 files. I don't want to receive 150 emails...